platform/windows/src/headless_backend_wgl.cpp

```diff
@@ -197,13 +197,19 @@ class WGLDisplayConfig {
     ~WGLDisplayConfig() { UnregisterClassA(renderingWindowClass.lpszClassName, renderingWindowClass.hInstance); }
 
     static std::shared_ptr<WGLDisplayConfig> create() {
```

Comment:
Could we not refactor this whole method like so:
```cpp
static WGLDisplayConfig& create() {
    static auto instance = WGLDisplayConfig{Key{}};
    return instance;
}
```

C++ 11 added the whole thread-safe static trick so this should be functionally the same.

Reply:
Much nicer solution, thanks!

Reply:
@mwilsnd @louwers Could someone please re-run the failed jobs? The temporary node files download problem strikes again.